07-routing-gossip.md

```diff
@@ -28,49 +34,50 @@ The willingness of the initiating node to announce the channel is signaled durin
 
 ### Requirements
 
-If the `open_channel` message had the `announce_channel` bit set, then both nodes must send the `announcement_signatures` message, otherwise they MUST NOT.
-
-If sent, `announcement_signatures` messages MUST NOT be sent until `funding_locked` has been sent and the funding transaction has at least 6 confirmations.
+The `announcement_signatures` message is created by constructing a `channel_announcement` message corresponding to the newly established channel and signing it with the secrets matching an endpoint's `node_id` and `bitcoin_key`. The message is then sent using an `announcement_signatures`.
```

Comment:
...message, corresponding...channel, and...

Reply:
Yep, clearer. Fixed.